tests/integration/views/test_view_submission.py

```diff
@@ -47,18 +44,20 @@ def test_view_submission(self):
         self.assertInPage('Please check your answers carefully before submitting.')
 
         # Submit answers
-        self.table.put_item = Mock(return_value={'ResponseMetadata': {'HTTPStatusCode': 200}})
-        self.post(action=None)
+        with mock.patch('app.data_model.submitted_responses.put_item',
+                        return_value={'ResponseMetadata': {'HTTPStatusCode': 200}}) as m_put_item:
+            self.post(action=None)
 
-        item = self.table.put_item.call_args[1]
+            item = m_put_item.call_args[0][0]
```

Comment:
anything wrong with just `m_put_item` i see the intention behind it but it brakes consistency

Reply:
If we use `m_put_item` outside of the patch context manager then we'll need to indent the following patch inside of it.

I.e.
```
with mock.patch('put_item') as m_put_item:
    ...stuff
    with mock.patch('get_item', return_value=item):
        ...stuff
```

Reply:
i'm not sure I follow.